src/core/connectors.js

```diff
@@ -1279,5 +1279,9 @@ define(function() {
 		label: 'Funkwhale',
 		matches: [''],
 		js: ['connectors/funkwhale.js'],
+	}, {
+		label: 'R-a-dio',
+		matches: ['*://r-a-d.io/*', '*://www.r-a-d.io/*'],
```

Comment:
```suggestion
		matches: ['*://r-a-d.io/*'],
```

www redirects to apex domain so this isn't needed.